docs/spec.md

```diff
@@ -297,6 +298,29 @@ Informs the client about updates to the connection graph. This is only sent to c
 }
 ```
 
+### Service call failure
```

Comment:
Seems like we capitalize these in some other places below (I didn't check all of them)
```suggestion
### Service Call Failure
```